source/stylist/fortran.py

```diff
@@ -200,6 +203,34 @@ def examine_fortran(self, subject: FortranSource) -> List[Issue]:
         return issues
 
 
+class LabelledExit(FortranRule):
+    """
+    Catches cases where a construct is exited but not explicitly named.
+    """
+
+    def examine_fortran(self, subject: FortranSource) -> List[Issue]:
+        issues = []
+
+        for exit in subject.find_all(Fortran2003.Exit_Stmt):
+            if exit.items[1] is None:
+                issues.append(Issue('Usage of "exit" without label '
+                                    'indicating '
+                                    'which do construct is being exited from.',
```

Comment:
Throughout change unquoted references to "do" construct are hard to read. They don't scan well. This is particularly noticeable in output strings such as this and documentation strings.

Reply:
Agreed - All now read "do"